ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/index.scss

```diff
@@ -0,0 +1,53 @@
+.tooltip {
```

Comment:
I feel like because our CSS is global, the name of the class should match the name of the component to avoid naming collisions. Is it possible for this to be `.edit-gas-tooltip` instead? Or should this be somewhere else?

Reply:
agreed!